lib/scoped_search/query_builder.rb

```diff
@@ -246,6 +246,16 @@ def sql_test(field, operator, value, lhs, &block) # :yields: finder_option_type,
       end
     end
 
+    def find_has_many_through_association(field, through)
+      middle_table_association = nil
+      field.klass.reflect_on_all_associations(:has_many).each do |reflection| 
+        class_name = reflection.options[:class_name].downcase.pluralize if reflection.options[:class_name]
```

Comment:
`constantize.table_name` might be more accurate than `downcase.pluralize` for CamelCase class names, which should be 'camel_cases' rather than 'camelcases' I think.